src/main/java/org/wso2/maven/humantask/artifact/util/FileManagementUtils.java

```diff
@@ -141,28 +89,29 @@ static private void addToZip(String path, String srcFile, ZipOutputStream zip) {
 
 	static private void addFolderContentsToZip(String srcFolder, ZipOutputStream zip) {
 		File folder = new File(srcFolder);
-		String fileListe[] = folder.list();
+		String fileListArray[] = folder.list();
 		int i = 0;
-		if (fileListe != null) {
-			while (i < fileListe.length) {
-				addToZip("", srcFolder + File.separator + fileListe[i], zip);
+		if (fileListArray != null) {
+			while (i < fileListArray.length) {
+				addToZip("", srcFolder + File.separator + fileListArray[i], zip);
```

Comment:
Fixed in https://github.com/wso2/maven-tooling-bps/pull/7/commits/43199d0a0b82d1b130de3da85e294ab25d422fd2